Documentation~/TableOfContents.md

```diff
@@ -15,7 +16,7 @@
   * [Motion Blur](Motion-Blur)
   * [Screen Space Reflections](Screen-Space-Reflections)
   * [Vignette](Vignette)
-* Scripting
-    * [Manipulating the Stack](Manipulating-the-Stack)
-    * [Writing Custom Effects](Writing-Custom-Effects)
-* [Troubleshooting](Troubleshooting)
+* Scripting reference
```

Comment:
We use the term "Scripting Reference" for our API docs. As these are not API docs, I would suggest another heading here. I think maybe "Scripting" was OK, or something else that means "interacting with the PPv2 using scripts rather than the Editor UI".

Reply:
I'll revert it to just "Scripting" then.